spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1631,4 +1631,16 @@ An `InterceptionFactory` can be obtain be calling `BeanManager.createInterceptio
 
 The container must provide a bean with scope `@Dependent`, bean type `InterceptionFactory` and qualifier `@Default` allowing producer methods to obtain this factory to produce their instance.
 
+Following example demonstrates producer method definition using `InterceptionFactory`. Interceptor using `ActionBinding` interceptor binding will be associated with produced bean:
```

Comment:
Yes you can but for the purpose of this example it's ok I guess. I wanted to emphasize that the binding is de facto the association which isn't so obvious to me from your proposal.